release.toml

```diff
@@ -1,4 +1,3 @@
-no-dev-version = true
```

Comment:
GH won't let me add a suggestion but I've switched this to `dev-version = false` in other projects, which retains the current behaviour.